Selenium.AutomatedTests/AutomationScenarioBuilder.cs

```diff
@@ -78,6 +84,16 @@ public AutomationScenarioBuilder WaitUntilVisible(By elementSelector, Action<IWe
             return this;
         }
 
+        /// <summary>
+        /// Enqueues step that sleeps number of <paramref name="milliseconds"/>
+        /// </summary>
+        /// <returns>The same <see cref="AutomationScenarioBuilder"/> instance</returns>
+        public AutomationScenarioBuilder Wait(int milliseconds)
+        {
+            Thread.Sleep(milliseconds);
```

Comment:
For the moment it's fine. However we should provide a way to do it using async/await to don't block the current Thread.